content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -1,73 +1,163 @@
 
-*Lead Scoring* lets you rank and prioritize leads based on selected criteria. A higher score
-indicates that a lead is more likely to bring value to your company.
+The *Odoo CRM* app can automatically assign leads/opportunities to sales teams and salespeople. A
+standard practice is to assign leads based on the probability of winning each lead. Companies can
+prioritize the leads that are more likely to result in successful deals by quickly assigning them
+to the appropriate salespeople.
 
-For example, if customers from Canada tend to be more likely to purchase your products, you can
-assign them a higher score than customers from other countries.
+Odoo automatically calculates the probability of winning each lead using a method called *predictive
+lead scoring*.
 
-Configuration
-=============
+Predictive lead scoring
+=======================
+
+Predictive lead scoring is a machine-learning model that uses historical data from *Odoo CRM* to
+score open leads/opportunities. As a company processes opportunities through the CRM pipeline, Odoo
+collects data on which opportunities are won and lost. Predictive lead scoring uses this data to
+predict the probability of winning each new lead or opportunity. The more opportunities that are
+sent through the CRM pipeline, the more data Odoo collects, and the more accurate the probabilities
+become.
+
+Specifically, Odoo's predictive lead scoring uses the *naive Bayes* probability model:
+
+.. math::
+   \begin{equation}
+   P(A | B) = \frac{P(A) \times P(B | A)}{P(B)}
+   \end{equation}
 
-To use scoring, go to :menuselection:`Apps`, remove the :guilabel:`Apps` filter, search for
-:guilabel:`Lead Scoring` and then install the module.
+The probability of success of each opportunity is displayed on the opportunity form.
 
-.. image:: lead_scoring/lead-scoring-module.png
+.. image:: lead_scoring/probability-opportunity-form.png
    :align: center
-   :alt: Lead Scoring module installation
+   :alt: The probability of success displayed on the opportunity form.
 
-Create scoring rules
-====================
+The probability of success updates automatically as the opportunity progresses through the CRM
+pipeline. When an opportunity moves to the next stage, its probability of success automatically
+increases according to the predictive lead scoring algorithm.
 
-To manage scoring rules, go to :menuselection:`CRM --> Leads --> Scoring Rules`.
+Configuration
+-------------
 
-Customize your rules to fit the needs of your business, and add as many criteria as you want.
+Predictive lead scoring is always active in *Odoo CRM*. However, the variables used to calculate the
+probability of success can be customized in the settings.
 
-.. image:: lead_scoring/scoring-example.png
-   :align: center
-   :alt: Score Rule example
+To customize the variables used by predictive lead scoring, go to :menuselection:`CRM -->
+Configuration --> Settings`. Under :guilabel:`Predictive Lead Scoring`, click on the
```

Comment:
*Quick Note: Since "Predictive Lead Scoring" is capitalized here, it might be worth considering making it capitalized throughout - for uniformity sake. What do you think?

Reply:
I thought about doing this, however "predictive lead scoring" is a method used by other CRM companies as well, rather than a feature unique to (or invented by) Odoo, so I thought it best to keep it lowercase.